File: sdks/thunkmetrc-java/wrapper/src/main/java/io/github/thunkier/thunkmetrc/wrapper/models/Patient.java
package io.github.thunkier.thunkmetrc.wrapper.models;
import com.fasterxml.jackson.annotation.JsonProperty;public class Patient {
    @JsonProperty("HasSalesLimitExemption")
    public Boolean hasSalesLimitExemption;
    @JsonProperty("LicenseEffectiveEndDate")
    public String licenseEffectiveEndDate;
    @JsonProperty("LicenseEffectiveStartDate")
    public String licenseEffectiveStartDate;
    @JsonProperty("LicenseNumber")
    public String licenseNumber;
    @JsonProperty("OtherFacilitiesCount")
    public Integer otherFacilitiesCount;
    @JsonProperty("PatientId")
    public Integer patientId;
    @JsonProperty("RecommendedPlants")
    public Integer recommendedPlants;
    @JsonProperty("RecommendedSmokableQuantity")
    public Double recommendedSmokableQuantity;
    @JsonProperty("RegistrationDate")
    public String registrationDate;
}

File: sdks/thunkmetrc-kotlin/wrapper/src/main/kotlin/io/github/thunkier/thunkmetrc/wrapper/models/Patient.kt
package io.github.thunkier.thunkmetrc.wrapper.models

import com.fasterxml.jackson.annotation.JsonProperty



data class Patient(
    @JsonProperty("HasSalesLimitExemption")
    val hasSalesLimitExemption: Boolean? = null,
    @JsonProperty("LicenseEffectiveEndDate")
    val licenseEffectiveEndDate: String? = null,
    @JsonProperty("LicenseEffectiveStartDate")
    val licenseEffectiveStartDate: String? = null,
    @JsonProperty("LicenseNumber")
    val licenseNumber: String? = null,
    @JsonProperty("OtherFacilitiesCount")
    val otherFacilitiesCount: Int? = null,
    @JsonProperty("PatientId")
    val patientId: Int? = null,
    @JsonProperty("RecommendedPlants")
    val recommendedPlants: Int? = null,
    @JsonProperty("RecommendedSmokableQuantity")
    val recommendedSmokableQuantity: Double? = null,
    @JsonProperty("RegistrationDate")
    val registrationDate: String? = null
)
